### PR TITLE
Rephrasing documentation for image and file pipelines

### LINKED ISSUE #4034: Rephrase image pipeline documentation
## Summary
Clarity of the pipeline documentation could be improved by more clearly calling out the steps that need to be taken to enable a custom pipeline. I was encouraged to issue this request in my Reddit thread where I was struggling to resolve this issue: [thread](https://www.reddit.com/r/scrapy/comments/d6n07x/struggling_rename_images/). 

Existing documentation: 
https://docs.scrapy.org/en/latest/topics/media-pipeline.html
> To enable your media pipeline you must first add it to your project item pipeline setting.
> For Images Pipeline, use:
> ITEM_PIPELINES = {'scrapy.pipelines.images.ImagesPipeline': 1}

To me, it reads like "to use this custom pipeline, just add this line to your settings.py file". If the doc maintainers wanted to be more clear, it could have read "add this line to your item pipeline settings and replace these fields with the appropriate fields from your project", like I do below:

> ITEM_PIPELINES = {f'{your_scrapername}.pipelines.{your_item_pipeline_class_name}: 1*}*

## Motivation

Improve clarity/readability, help newer users get up to speed more quickly. 

## Describe alternatives you've considered
on reddit /u/Gallaecio/ suggested the following as an alternative: 

> I think you might have a point. I bet something as simple as changing 
> scrapy.pipelines.images.ImagesPipeline to something like 
> my_project.pipelines.MyImagesPipeline would make things more obvious.

Either solution would help to show that 'scrapy' references the specific project you're working on, and 'ImagesPipeline' is the custom class you setup for your project. 

## Additional context
In my attempts to resolve my own problem I found many threads going back years on stack overflow and reddit that highlighted the same issue. I believe that the change would help new users avoid a potentially frustrating issue.

### PR DESCRIPTION
**Pull request by 1um0s**
Rephrasing documentation for image and file pipelines

Fixes #4034

### CODE REVIEW

File: docs/topics/media-pipeline.rst [hunk 1]
```diff
@@ -86,17 +86,15 @@ Enabling your Media Pipeline
 .. setting:: IMAGES_STORE
 .. setting:: FILES_STORE
 
-To enable your media pipeline you must first add it to your project
-:setting:`ITEM_PIPELINES` setting.
+To enable your media pipeline you must add its class to the :setting:`ITEM_PIPELINES` setting, like in the following example:
```
Comment (Gallaecio) [inline docs/topics/media-pipeline.rst]: Unless I’m missing something, it should be more like:

```suggestion
To enable your media pipeline you must add its class import path to the :setting:`ITEM_PIPELINES` setting, like in the following example:
```

And the code changes below must retain the quotes (e.g. `my_project.pipelines.MyImagesPipeline` → `'my_project.pipelines.MyImagesPipeline'`)

**Comment by elacuesta** [comment]
I think this could be a bit misleading. As I see it, it reads as if you **need** to have your own pipeline (either subclassed or written from scratch), since it takes the class from the `myproject`. However, the built-in media pipelines do work out of the box, as fair as I remember. Perhaps this change could be applied to the "Extending the Media Pipelines" section instead.

**Comment by 1um0s** [comment]
@elacuesta - Okay, I see how it can be confusing now. Moved it to the "Extending the Media Pipelines" section.

**Comment by elacuesta** [comment]
There seem to be some unrelated changes to `scrapy/exporters.py`. Could you rebase your branch on top of the current `master`?

**Comment by 1um0s** [comment]
Sure. Made the change.
@elacuesta  - Difference was due to the reason outlined in https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch

File: docs/topics/media-pipeline.rst [hunk 2]
```diff
@@ -578,4 +577,12 @@ above::
             item['image_paths'] = image_paths
             return item
 
+
+To enable your custom media pipeline component you must add its class to the
```
Comment (elacuesta) [inline docs/topics/media-pipeline.rst]:
```suggestion
To enable your custom media pipeline component you must add its class import path to the
```

as @Gallaecio suggested in https://github.com/scrapy/scrapy/pull/4252#discussion_r360880296

**Comment by kmike** [comment]
Thanks @1um0s for the fix, and @Gallaecio and @elacuesta for the reviews!

**Pull request merged**